src/pymodaq/daq_logger.py

```diff
@@ -403,21 +407,55 @@ def queue_command(self, command):
             self.stop_scan_flag = True
             self.stop_logging()
 
-    def do_save_continuous(self, datas):
-        """
+    @Slot(OrderedDict)
+    def do_save_continuous(self, acquisition_from_viewer):
```

Comment:
Maybe a typehint could be added to `acquisition_from_viewer`

Reply:
Done!

Reply:
Here cd169f35d12eae3e0ae9ed0f6d162853d90264d7